doc/shapes.rst

```diff
@@ -143,20 +143,20 @@ Sphere unions
 
 Type: ``SphereUnion``
 
-A collection of spheres, defined by their radii and positions.
+A collection of spheres, defined by their diameters and positions.
```

Comment:
```suggestion
A collection of spheres, defined by their diameters and centers.
```

Reply:
Also, the header of the section should probably in caps (`Sphere Unions`), following the convention for he other shapes.